nrm_django/grants/forms.py

```diff
@@ -0,0 +1,68 @@
+import datetime
+import uuid
+
+from django import forms
+from django.core.exceptions import ValidationError
+
+from .models import Grant
+
+
+class GrantForm(forms.ModelForm):
+    class Meta:
+        model = Grant
```

Comment:
Love the way this makes creation easy to connect up with the existing list view.

Reply:
Modelforms + Generic views are just amazing.